docs/DeveloperGuide.md

```diff
@@ -288,24 +295,19 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`    | Teaching Assistant                                        | display the possible actions that  I can take, or do                                   | have a quick reference without the need of reading a user guide                               |
 | `* *`    | Teaching Assistant                                        | edit the details of the students                                                       | fix any errors that occurred during the insertion of student details                          |
 | `* *`    | Teaching Assistant who is teaching in multiple semesters  | archive the existing data                                                              | save a copy on the local machine before undergoing a complete deletion of data                |
```

Comment:
I feel "archive" should be a must-have since it is abit pointless for the user to have a new set of save file after each session.

Reply:
Upped "archive" priority to must have.